### PR TITLE
Feat/3828  activity history

### PR DESCRIPTION
**Pull request by asf-algoo**
See #3828.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ DropdownMenu.propTypes = {
   buttonDisabled: PropTypes.bool,
   buttonIcon: PropTypes.string,
   buttonImage: PropTypes.string,
-  buttonLabel: PropTypes.string,
+  buttonLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.component]),
```
Comment (gbonaspetti) [inline frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx:53]: You could have used buttonOpts instead of buttonLabel, but this works too.

**Review by gbonaspetti — APPROVED** [review]

File: frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx (hunk 1 above)
Comment (grignards) [reply frontend_lib/src/component/DropdownMenu/DropdownMenu.jsx:53]: Yes, I'll do that.

**Comment by gbonaspetti** [comment]
In small screens we don't have de type
Screenshot from 2020-12-01 09-55-35
the type of modification should be displayed in the menu even in small screens (only in the menu, not in the activity)

**Comment by gbonaspetti** [comment]
Tested on branch

**Pull request merged**